app/AppComponents.scala

```diff
@@ -29,8 +29,8 @@ trait AWSComponent { self: BuiltInComponents =>
 
 trait ControllersComponent {
   self: BuiltInComponents with AWSComponent =>
-  def messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
-  def appController = new Application(dynamoClient, talksTableName)
+  val messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
```

Comment:
Originally we changed them out of desperation as we couldn't work out what had broken but they should be def (or lazy val) 😃